### PR TITLE
Autogenerate search and kyc-documents resources

### PR DESCRIPTION
**Pull request by agualis**

### CODE REVIEW

**Review by adamaltman — DISMISSED** [review]

File: src/resources/kyc-documents-resource.js [hunk 1]
```diff
@@ -1,46 +1,34 @@
...
-        matches({id, data}) {
-            return apiHandler.post(`kyc-documents/${id}/matches`, data);
-        },
-    };
-};
+export default function KycDocumentsResource({apiHandler}) {
+  return {
+    getAll({limit = null, offset = null, filter = null, sort = null} = {}) {
+      const params = {limit, offset, filter, sort};
+      return apiHandler.getAll(`kyc-documents`, params);
```
Comment (rensftw) [inline src/resources/kyc-documents-resource.js:10]: `expand` and `q` are missing, is this ok?

File: src/resources/kyc-documents-resource.js [hunk 2]
```diff
@@ -1,46 +1,34 @@
...
+    },
+    create({id = '', data}) {
+      return apiHandler.create(`kyc-documents/${id}`, id, data);
+    },
+    get({id}) {
+      return apiHandler.get(`kyc-documents/${id}`);
+    },
+    update({id, data}) {
+      return apiHandler.put(`kyc-documents/${id}`, data);
+    },
```
Comment (rensftw) [inline src/resources/kyc-documents-resource.js:20]: `expand` is missing in the params of these three methods, is this ok?

File: src/resources/search-resource.js [hunk 3]
```diff
@@ -5,8 +5,8 @@
 
 export default function SearchResource({apiHandler}) {
   return {
-    get({sort, limit, offset, q, expand = null}) {
-      const params = {sort, limit, offset, q, expand};
+    get({sort = null, limit = null, offset = null, q = null}) {
```
Comment (rensftw) [inline src/resources/search-resource.js:8]: `expand` is missing here too, do we even use that parameter?

**Review by rensftw — COMMENTED** [review]
LGTM, but there are a few parameters missing in the autogenerated code 👇🏻

File: src/resources/kyc-documents-resource.js (hunk 2 above)
Comment (agualis) [reply src/resources/kyc-documents-resource.js:20]: We don't need `expand`. This PR fixes that discrepancy.

File: src/resources/search-resource.js (hunk 3 above)
Comment (agualis) [reply src/resources/search-resource.js:8]: We don't need `expand`. This PR fixes that discrepancy.

**Review by rensftw — APPROVED** [review]

**Pull request merged**